### PR TITLE
Support the major Blender fork Bforartists

### PR DESCRIPTION
**Pull request by devesnosy**
[Bforartists](https://www.bforartists.de/) is one of the few major Blender forks out there, it has been around for more than 8 years and maintained its own user base,

this PR was sponsored by @Draise14 to support listing, downloading and launching of Bforartists releases,
the original idea was to fork it as its own launcher,

but personally I immediately saw, even before starting working on it, that it is more suitable to merge it into the main project, since the change is very small anyways,

and this PR indeed reflects that, the changes are minimal and don't affect the Blender side of the code.

blender_launcher_bfa

### CODE REVIEW

**Review by zeptofine — COMMENTED** [review]
Very nice! I just have one question about the `username` in the scrape_bfa_releases function. I'm not really familiar with webdav :p

File: source/threads/scraper.py [hunk 1]
```diff
@@ -439,3 +448,40 @@ def scrap_stable_releases(self):
 
         r.release_conn()
         r.close()
+
+    def scrape_bfa_releases(self):
+        https_url = "https://cloud.bforartists.de/index.php/s"
+        webdav_url = "https://cloud.bforartists.de/public.php/webdav"
+        username = "JxCjbyt2fFcHjy4"
```
[the remark this answers is not in the document]
Comment (devesnosy) [reply source/threads/scraper.py:455]: this is a public value not user specific, fine to keep it in public like this, and yep can be shown as constant

File: source/widgets/base_build_widget.py [hunk 2]
```diff
@@ -38,6 +38,9 @@ def show_release_notes(self):
             v = re.sub(r"\D", "", str(self.build_info.semversion.finalize_version()))
 
             webbrowser.open(f"https://www.blender.org/download/lts/#lts-release-{v}")
+        elif self.build_info.branch == "bforartists":
```
Comment (zeptofine) [inline source/widgets/base_build_widget.py:41]: How is this going? I think the easiest route would just be to link here: https://www.bforartists.de/release-notes/

Comment (devesnosy) [reply source/widgets/base_build_widget.py:41]: yep working on it, I just wanted it to link to the correct per version release notes directly, but indeed I agree with you, if I wasn't able to get direct link to per version release notes I'll use the link, sadly the bforartists de release notes link don't have correct HTML element tag IDs so we can have per version URLs so my current thinking is NextCloud release notes text files

**Comment by devesnosy** [comment]
Apologies for delay now it is done
https://github.com/Victor-IX/Blender-Launcher-V2/pull/151/commits/c271e51f4b25a30f74987bbcd696195f6d4c951a

**Pull request merged**